gcexport.py

```diff
@@ -732,7 +799,7 @@ def export_data_file(activity_id, activity_details, args, file_time, append_desc
                     unzipped_name = zip_obj.extract(name, directory)
                     # prepend 'activity_' and append the description to the base name
                     name_base, name_ext = splitext(name)
-                    new_name = directory + '/activity_' + name_base + append_desc + name_ext
+                    new_name = directory + sep + prefix + 'activity_' + name_base + append_desc + name_ext
```

Comment:
```suggestion
                    new_name = join(directory, f'{prefix}activity_{name_base}{append_desc}{name_ext}
```

Reply:
Python2 compatibility

```suggestion
                    new_name = join(directory, prefix + 'activity_' + name_base + append_desc + name_ext)
```